static/extensions/justablock/gitpenguin.js

```diff
@@ -125,13 +129,31 @@
               }
             }
           },
+          {
+            blockType: Scratch.BlockType.BUTTON,
+            func: "showHiddenBlocks",
+            text: Scratch.translate("Show dangerous blocks using a github token")
+          },        
         ],
         menus: {
           TOGGLE: ["on", "off"]
         }
       };
     }
 
+    showHiddenBlocks() {
+      alert(Scratch.translate("Anyone with your github token can access your account and create repositories, delete repositories, commit changes, and more, all while pretending to be you. You should not include your token in any capacity in a project shared with other people."))
+      const response = prompt(Scratch.translate("To show these blocks, type \"I will not post a project with my github token anywhere and I understand people can impersonate me if I do\" (not case sensitive)."))
```

Comment:
Maybe use `ScratchBlocks.prompt` instead? It is a bit complicated to work with, so here I did the liberty of turning it into something that can replace the regular prompt function, as long as the function it gets used in is an async function:

```js
var response = await new Promise((res, _) => ScratchBlocks.prompt("...", "", (answer) => { res(answer) }, "Show potentially dangerous blocks", "broadcast_msg"));
```

`broadcast_msg` is used here to make sure it only shows the text field, and no radio buttons. Only thing that should be replaced is the "...", which should be replaced with the prompt string.